src/Compilers/VisualBasic/Portable/Lowering/LambdaRewriter/LambdaRewriter.vb

```diff
@@ -907,11 +918,65 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return result
         End Function
 
+        Private Sub GetLambdaId(syntax As SyntaxNode, closureKind As ClosureKind, closureOrdinal As Integer, ByRef topLevelMethodId As MethodDebugId, ByRef lambdaOrdinal As Integer)
+            Debug.Assert(syntax IsNot Nothing)
+
+            Dim lambdaOrLambdaBodySyntax As SyntaxNode
+            Dim isLambdaBody As Boolean ' indicates whether a node needs mapping for lambda or its bodies
```

Comment:
If is somewhat unclear when a node needs a mapping. Consider adding a comment explaining conditions when the mapping is needed.